### PR TITLE
Refactor speed test

### PR DESCRIPTION
**Pull request by mustafaboleken**
https://github.com/ant-media/conference-call-application/issues/358

This pull request removes the mechanism to publish a stream before making play speed test for the play only users.
Instead of publishing and playing in the client side, it tries to play a "static" streamId which is called **speedTestSampleStream** in the same Ant Media Server application. Using a static stream reduces the load on the server side in case there are so much new play only users. I will add a mechanism to add a playlist and use the **speedTestSampleStream** streamId if there is no stream with id **speedTestSampleStream**

### CODE REVIEW

File: react/src/pages/AntMedia.js [hunk 1]
```diff
@@ -755,6 +746,72 @@ function AntMedia(props) {
         stopSpeedTest();
     }
 
+    function calculateThePlaySpeedTestResult() {
```
Comment (mustafaboleken) [inline react/src/pages/AntMedia.js:749]: @mekya can you please take a look at the calculation here?

File: react/src/__tests__/pages/AntMedia.test.js [hunk 2]
```diff
@@ -1841,7 +1841,7 @@ describe('AntMedia Component', () => {
     });
 
     await waitFor(() => {
-      expect(webRTCAdaptorPlaySpeedTestConstructor).not.toBe(undefined);
+      expect(webRTCAdaptorPlaySpeedTestConstructor).toBe(undefined);
```
Comment (burak-58) [inline react/src/__tests__/pages/AntMedia.test.js:1844]: remove unused `webRTCAdaptorPlaySpeedTestConstructor`

File: react/src/pages/AntMedia.js [hunk 3]
```diff
@@ -756,20 +736,106 @@ function AntMedia(props) {
...
+        // Calculate the time difference (in seconds)
+        let timeElapsed = (stats.currentTimestamp - stats.startTime) / 1000; // Convert ms to seconds
+
+        // Calculate incoming bitrate (bits per second)
+        let bytesReceivedDiff = stats.lastBytesReceived - stats.firstBytesReceivedCount;
+        let incomingBitrate = (bytesReceivedDiff * 8) / timeElapsed; // Convert bytes to bits
+
+        // Calculate packet loss
+        let videoPacketsLost = stats.videoPacketsLost;
+        videoPacketsLost = (videoPacketsLost < 0) ? 0 : videoPacketsLost;
```
Comment (burak-58) [inline react/src/pages/AntMedia.js:759]: remove unnecesaary negative checks

**Review by burak-58 — APPROVED** [review]

File: react/src/pages/AntMedia.js (hunk 3 above)
Comment (mustafaboleken) [reply react/src/pages/AntMedia.js]: it's removed

File: react/src/__tests__/pages/AntMedia.test.js (hunk 2 above)
Comment (mustafaboleken) [reply react/src/__tests__/pages/AntMedia.test.js]: it's removed

**Pull request merged**